modules/installation-installer-provisioned-vsphere-config-yaml.adoc

```diff
@@ -21,69 +21,94 @@ ifeval::["{context}" == "installing-restricted-networks-vmc"]
 endif::[]
 
 [id="installation-installer-provisioned-vsphere-config-yaml_{context}"]
-= Sample install-config.yaml file for an installer-provisioned VMware vSphere cluster
+= Sample `install-config.yaml` file for an installer-provisioned VMware vSphere cluster
```

Comment:
A note should be added that the pre 4.13 platform spec still functions

Reply:
This would mean that one could create a config file and not define failure domains?

The following paragraph is not accurate?

> You must specify at least one failure domain for your {product-title} cluster, so that {product-title} can evenly spread virtual machine nodes across domains or groups on a vCenter server. Consider specifying multiple failure domains if you need to provision virtual machine nodes in different datacenters, clusters, datastores, and so on.

Reply:
Hi @jcpowermac . I added the following note:

> The {product-title} {product-version} release introduces two `spec:` fields: `vcenters` and `failureDomains` for the `install-config.yaml` file. 
You can ignore these fields by not setting values for them. This means that you are using the deprecated configuration and you might receive error messages.

Can you clarify where a product use might see the error message? On the vSPhere web console or the oc CLI?